x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/tests/index.ts

```diff
@@ -8,7 +8,7 @@
 import { FtrProviderContext } from '../../../../ftr_provider_context';
 
 export default function ({ loadTestFile }: FtrProviderContext) {
-  describe('@ess @serverless SecuritySolution Saved Objects', () => {
+  describe('@ess @serverless @serverlessQA SecuritySolution Saved Objects', () => {
```

Comment:
@MadameSheema do we have documentation on how to test things for the periodic pipeline? I didn't find what I was looking for in this [README](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_api_integration/README.md)

Reply:
Take a look [here](https://docs.elastic.dev/security-solution/teams/analyst-experience/automation/serverless/README)